addons/rating/models/rating.py

```diff
@@ -53,6 +53,7 @@ def _default_access_token(self):
         'mail.message', string="Linked message",
         index=True, ondelete='cascade',
         help="Associated message when posting a review. Mainly used in website addons.")
+    internal = fields.Boolean('Employee Only', index=True)
```

Comment:
same comment as for message internal field

Reply:
Renamed to is_internal too then.